api-testing/functional/tests/cypress/integration/01-institution/patch-institutions-institution.ts

```diff
@@ -0,0 +1,36 @@
+describe("Institution api", () => {
+  const access_token =
```

Comment:
Is the token being manually added to the tests? The tests should run without manual intervention. The token should be acquired from the Keyclock and used in the tests in the same way that we authenticated the users.
We need to: 
1 - Execute an API call to Keycloak;
2 - Get the authentication token;
3 - Provide the token as a parameter to the API call.